scripts/chromatic-stories.js

```diff
@@ -0,0 +1,34 @@
+const fs = require('fs');
```

Comment:
Can we change the name of this script to something like `genetate-chromatic-stories`?

Reply:
Good idea. I added section in docs and renamed script.